src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,7 +2800,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-macos`
Get-Random.Should accept collection containing empty string for -InputObject
```powershell
Cannot validate argument on parameter 'InputObject'. The argument is null, empty, or an element of the argument collection contains a null value. Supply a collection that does not contain any null values and then try the command again.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Random.Tests.ps1: line 187
187:         1..10 | ForEach-Object {

```
Get-Random.Should accept $null in collection for -InputObject
```powershell
Cannot validate argument on parameter 'InputObject'. The argument is null, empty, or an element of the argument collection contains a null value. Supply a collection that does not contain any null values and then try the command again.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Random.Tests.ps1: line 193
193:         1..10 | ForEach-Object {

```